### PR TITLE
Update Categories & Product Names

### PR DESCRIPTION
**Pull request by axxroytovu**
Big update to standardize product category, subtype and name. 

Tied to https://github.com/mtgjson/mtg-sealed-content/issues/90 and https://github.com/mtgjson/mtgjson/issues/1139

### CODE REVIEW

File: data/contents/M12.yaml [hunk 1]
```diff
@@ -45,21 +45,21 @@ products:
   2012 Core Set Intro Pack Display:
     sealed:
     - count: 3
-      name: '2012 Core Set Intro Pack: Blood and Fire (Red/Black)'
+      name: '2012 Core Set Intro Pack Blood and Fire'
```
Comment (kodawah) [inline data/contents/M12.yaml:48]: do we still need the '' if there are no special chars in the product name?

File: data/products/FRF.yaml [hunk 2]
```diff
@@ -8,10 +8,10 @@ products:
     release_date: '2015-01-21'
```
Comment (kodawah) [inline data/products/FRF.yaml:8]: these should be split to their own commit I think (ping me on discord if you need help with git)

File: data/contents/M12.yaml (hunk 1 above)
Comment (axxroytovu) [reply data/contents/M12.yaml:48]: That will get cleaned up by the GitHub automation, not critical to resolve at this level.

File: data/contents/CMD.yaml [hunk 3]
```diff
@@ -1,44 +1,44 @@
 code: cmd
 products:
-  Commander Counterpunch Deck:
+  Commander (2011) Commander Deck Counterpunch:
```
Comment (kodawah) [inline data/contents/CMD.yaml:3]: do you think we could keep "Commander 2011" without the parenthesis? just to follow the same edition name verbatim

File: data/contents/CLB.yaml [hunk 4]
```diff
@@ -51,18 +51,18 @@ products:
   Commander Legends Battle for Baldurs Gate Commander Deck Display:
```
Comment (kodawah) [inline data/contents/CLB.yaml:51]: this is a commander product but the commit for commander product is kept separate, is that intentional?

File: data/contents/CMD.yaml (hunk 3 above)
Comment (axxroytovu) [reply data/contents/CMD.yaml:3]: Officially the set name is "Commander" and the 2011 is us making a note to clarify.

File: data/contents/CLB.yaml (hunk 4 above)
Comment (axxroytovu) [reply data/contents/CLB.yaml:51]: The only reason I made a separate commit was to keep track of what sets I'd reviewed already. I'm going to squash everything anyway.

File: data/products/FRF.yaml (hunk 2 above)
Comment (axxroytovu) [reply data/products/FRF.yaml:8]: Are you saying to pull this into an entire separate PR or just to split the commit? I was planning on squashing this PR after the review.

File: data/contents/CMD.yaml (hunk 3 above)
Comment (kodawah) [reply data/contents/CMD.yaml:3]: yes but that's not the edition name that every other set is based on

we should either stick with the official product name (Commander) or stick with the mtgjson/scryfall edition name (Commander 2011), not making up a new name strategy for this product IMO

File: data/products/FRF.yaml (hunk 2 above)
Comment (kodawah) [reply data/products/FRF.yaml:8]: no just split the commit, regardless if we squash so that history is preserved clean

(im personally not a fan of squash commits, it ties us to github, makes review/history blame harder)

File: data/contents/CMD.yaml (hunk 3 above)
Comment (axxroytovu) [reply data/contents/CMD.yaml:3]: If the Scryfall name is Commander 2011 I feel better about removing the parenthesis.

Comment (kodawah) [reply data/contents/CMD.yaml:3]: yup https://scryfall.com/sets/cmd :)
